src/setup/export_setup.py

```diff
@@ -0,0 +1,42 @@
+import os
+import time
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+kibana_username = "elastic"
+kibana_password = os.getenv("KIBANA_SYSTEM_PASSWORD")
+# consider rabbitmq has the same host as kibana
+host = os.getenv("RABBITMQ_HOST")
+url = f"http://{host}:16601/api"
+
+print("kibana_username: ", kibana_username)
+print("kibana_password: ", kibana_password)
+print("host: ", host)
+print("url: ", url)
+
+session = requests.Session()
+
+url2 = f"{url}/status"
+
+print(f"Waiting for kibana API to be up at {url2}...")
+while True:
+    try:
```

Comment:
May want to add some comments for clarity

Reply:
What's not clear? The print already tells what's happening

Reply:
Sorry wrong line, ment line 32-38 which aren't clear on a first read-through

Reply:
[Done](https://github.com/Integration-Project-Team-1/Monitoring/pull/2/commits/7293a5c83fba3eabc0a88bd52a2f4c8dc57d218f)